assets/scss/_styles_project.scss

```diff
@@ -568,6 +586,14 @@ END MARKETING CSS
     max-width: 40%;
   }
 
+  .hover-card.hover-card-xxs {
```

Comment:
I think you actually don't need any of this

Reply:
gotcha done!